stable/jenkins/values.yaml

```diff
@@ -6,7 +6,7 @@
 Master:
   Name: jenkins-master
   Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
-  ImageTag: "v0.6.0"
+  ImageTag: "v0.7.0"
```

Comment:
@jwhitcraft That image isn't publicly available. Would you be opening a separate PR/issue for that? That'd allow us to merge in the rest of the changes now.

```
$ docker pull gcr.io/kubernetes-charts-ci/jenkins-master-k8s:v0.7.0
Pulling repository gcr.io/kubernetes-charts-ci/jenkins-master-k8s
Tag v0.7.0 not found in repository gcr.io/kubernetes-charts-ci/jenkins-master-k8s
```